src/transport/raw/tests/TestTCP.cpp

```diff
@@ -159,14 +159,11 @@ class MockTransportMgrDelegate : public chip::TransportMgrDelegate
 
 /////////////////////////// Init test
 
-class TestTCP : public ::testing::Test
+class TestTCP : public ::testing::Test, public chip::Test::IOContext
 {
 protected:
-    TestTCP() { inContext = new TestContext(); }
-    ~TestTCP() { delete inContext; }
-    void SetUp() { ASSERT_EQ(inContext->Init(), CHIP_NO_ERROR); }
-    void TearDown() { inContext->Shutdown(); }
-    TestContext * inContext;
+    void SetUp() { ASSERT_EQ(this->Init(), CHIP_NO_ERROR); }
+    void TearDown() { this->Shutdown(); }
 };
 
 void CheckSimpleInitTest(TestContext * ctx, Inet::IPAddressType type)
```

Comment:
You can add these two functions to test context class then you do not have to pass context

Reply:
Done.